grunt/grunt.md

```diff
@@ -0,0 +1,23 @@
+OVERVEIW<br><br>
+Grunt,the javascript task runner,written in Node.js,is used to automate tasks that are performed in routine manner.Automation tools are used to lessen the workload by doing the assigned tasks faster with better precision.Grunt mainly helps in debugging code, concatenating and minifying CSS and JavaScript files, compressing images and applying changes to a server or files on the server.These tasks may sound simple but they may be time consuming,if we are working on large projects,upon using grunt such tasks can be performed easily and at a faster pace.<br><br>

```

Comment:
1) Remove comma after runner in the very first line! Else, everything seems perfect! :heart: